tests/chart_tests/test_nats_jetstream.py

```diff
@@ -182,3 +182,15 @@ def test_jetstream_hook_job_disabled(self, kube_version):
         )
 
         assert len(docs) == 0
+
+    def test_jetstream_job_with_scc_disabled(self, kube_version):
+        """Test that helm renders statefulset template for nats
+        jetstream with SA Disabled."""
+        docs = render_chart(
+            kube_version=kube_version,
+            values={},
+            show_only=[
+                "charts/nats/templates/jetstream-job-scc.yaml",
+            ],
+        )
+        assert len(docs) == 0
```

Comment:
We should add a test with scc and jetstream enabled, and scc and stan enabled. We can probably just modify this test https://github.com/astronomer/astronomer/blob/98951f41ec/tests/chart_tests/test_nats_jetstream.py#L149-L169